test/controllers/PenaltyDetailsController.test.ts

```diff
@@ -21,7 +21,7 @@ const config = getDefaultConfig();
 
 describe('PenaltyDetailsController', () => {
 
-    const navigation = {} as Navigation;
+    const navigation: Navigation = {} as Navigation;
```

Comment:
What do you think about making these kind of changes in separate PR so they do not distract from the purpose of PR / user story?

Reply:
oops, sorry this was me! i took note of the suggestions you made last week :-) i will remove these for now and we can add them back in the future